### PR TITLE
Fixes NMS-241

### PR DESCRIPTION
**Pull request by nmsyogesh**
Changes in USerControllerBundleIt.java, all committed cases passed

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: testing/src/test/java/org/motechproject/nms/testing/it/api/UserControllerBundleIT.java [hunk 1]
```diff
@@ -1479,5 +1496,538 @@ public void verifyFT184() throws IOException,
...
+	whitelistState = rh.delhiState();
+	stateDataService.create(whitelistState);
+
+	whitelistStateDataService.create(new WhitelistState(whitelistState));
+
+	nonWhitelistState = rh.karnatakaState();
+	stateDataService.create(nonWhitelistState);
+    }
+
+    HttpGet createHttpGetUserDetails(Service serviceType, String callingNo,
```
Comment (larubbio) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/UserControllerBundleIT.java]: Is there a reason the existing createHttpGet method wasn't used?

File: testing/src/test/java/org/motechproject/nms/testing/it/api/UserControllerBundleIT.java [hunk 2]
```diff
@@ -1479,5 +1496,538 @@ public void verifyFT184() throws IOException,
...
+		sb.append(seperator);
+	    }
+	    sb.append("callId=");
+	    sb.append(callId);
+	}
+	// System.out.println("Request url:" + sb.toString());
+	return RequestBuilder.createGetRequest(sb.toString());
+    }
+
+    HttpPost createHttpPostSetLanguageLocationCode(Service serviceType,
```
Comment (larubbio) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/UserControllerBundleIT.java]: Is there a reason the existing createHttpPost wasn't used?

Comment (nmsyogesh) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/UserControllerBundleIT.java]: Sorry, I was not aware of the methods. I have changed the methods and committed the code as per your instructions.

**Comment by larubbio** [comment]
ok to test

**Comment by larubbio** [comment]
@koshalt or @frankhuster Can one of you review this?  I removed ServiceUsage as a domain object since CallDetails had all the needed data.

**Pull request merged**